api/client/exec.go

```diff
@@ -17,6 +17,7 @@ import (
 // Usage: docker exec [OPTIONS] CONTAINER COMMAND [ARG...]
 func (cli *DockerCli) CmdExec(args ...string) error {
 	cmd := Cli.Subcmd("exec", []string{"CONTAINER COMMAND [ARG...]"}, Cli.DockerCommands["exec"].Description, true)
+	escapeKeys := cmd.String([]string{"-escapeKeys"}, "", "Set up escape key sequence")
```

Comment:
Yeah soon soon :sweat_smile: I'm adding the flag there for consistency now but a PR will be there soon :wink: